workshops/0_installation_party/README.md

```diff
@@ -17,54 +17,66 @@ Our goal is to get you set up with the base things you will need to participate
 
 ```sh
 Python 3.x.x
 ```
 
 Other wise follow [these](https://tutorial.djangogirls.org/en/installation/#python) instructions.
 
-*If the version is below 3.8.x then you will need to upgrade the version.* You can read about managing multiple python versions [here](https://github.com/pyenv/pyenv#simple-python-version-management-pyenv).
+> **Note**
+> If the version is below 3.8.x then you will need to upgrade the version.
+> You can read about managing multiple python versions [here](https://github.com/pyenv/pyenv#simple-python-version-management-pyenv).
 
 
 ## Setting up Virtual environments
 
-Virtual environments are isolated spaces on your computer where you can run your code and install libraries without affecting the rest of your machine's setup. We recommend setting up [python venv](https://docs.python.org/3/library/venv.html). Convention is to create a new one for each project to help avoid conflicts with other instatllations. Virtual environments are easy to delete and rebuild incase of errors. [Read more](https://realpython.com/python-virtual-environments-a-primer/).
+Virtual environments are isolated spaces on your computer where you can run your code and install libraries without affecting the rest of your machine's setup. We recommend setting up [python venv](https://docs.python.org/3/library/venv.html). You will see that there are many ways to create these, and even with different options, the idea of using `venv` is because it's included with Python already.
+
+Convention is to create a new one for each project to help avoid conflicts with other installations. Virtual environments are easy to delete and rebuild in case of errors. [Read more](https://realpython.com/python-virtual-environments-a-primer/).
 
 ## Setting up a Code editor (IDE)
 
-There are many code editors and IDEs (Integrated Development Environment), which one you choose to use for viewing, editing, updating, running and debugging your code is up to you. We recommend using [virtual studio code](https://code.visualstudio.com) because it is popular among professional programmers, offers multiple languages support and is free to use. [Read more](https://realpython.com/python-ides-code-editors-guide/).
+There are many code editors and IDEs (Integrated Development Environment), which one you choose to use for viewing, editing, updating, running and debugging your code is up to you.
+We recommend using [Visual Studio Code (VSC)](https://code.visualstudio.com) because it is popular among professional programmers, offers multiple languages support and is free to use. [Read more](https://realpython.com/python-ides-code-editors-guide/).
+
+In case you prefer to install another IDE, it's totally fine, and at the end
+will depend of your taste.
 
 ## Installing Git
 
 Git is a version control system which allows us to track changes in computer files. Files saved in Git can be "rewound" to find previous versions, which can be very useful when debugging!
 
-First! 
```

Comment:
the extra two spaces here ensure in markdown that the next part is on a new line, but it doesn't really matter.

Reply:
sorry, automatically my editor removes all the trailing spaces, so this got affected.